frontend/src/pages/Admin/Components/Broadcast/AddBroadcasts/AddBroadcasts.jsx

```diff
@@ -70,17 +84,54 @@ export function AddBroadcasts() {
       tagRef.current.value = "";
     }
   };
+  const addImageUrl = () => {
+    const url = imageRef.current.value;
+    if (url.trim()) {
+      setImageUrls(prevUrl => [...prevUrl, url.trim()]);
+      setFormData({ ...formData, imageUrl: [...formData.imageUrl, url.trim()] });
+      imageRef.current.value = "";
+    }
+  }
 
   const removeTag = tag => {
     setTags(tags.filter(t => t !== tag));
     setFormData({ ...formData, tags: formData.tags.filter(t => t !== tag) });
   };
-
+  const removeImageUrl = url => {
+    setImageUrls(imageUrls.filter(u => u !== url));
+    setFormData({ ...formData, imageUrl: formData.imageUrl.filter(u => u !== url) });
+  }
+  async function uploadData(formData) {
+    try {
+      let url = `${END_POINT}/broadcast`
+      const response = await fetch(url, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${localStorage.getItem('token')}`
+        },
+        body: JSON.stringify(formData)
+      });
+      const data = await response.json();
+      setIsUploadingData(false)
+      setToastMessage("broadcats added successfully!")
+      setOpenToast(true)
+      console.log(data)
+    }
+    catch (err) {
+      console.log(err)
+      setIsUploadingData(false)
+      setToastMessage("Something went wrong");
+      setOpenToast(true)
+      setSeverity("error")
+    }
+  }
   const onSubmit = e => {
```

Comment:
After successful submission of the form, the form value should reset.
If any error occurs, it shouldn't reset.

Reply:
- fixed toasts status on error response
- Resetable form

https://user-images.githubusercontent.com/74807466/166094348-9ef0faef-0c53-4ecc-8dd7-df1443ccdee8.mp4